cg/meta/archive/archive.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from collections import namedtuple
+from typing import Dict, List
+
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.meta.archive.ddn_dataflow import DDNDataFlowApi
+from cg.store import Store
+
+LOG = logging.getLogger(__name__)
+
+PathAndSample = namedtuple("PathAndSample", "path sample_internal_id")
+
+
+class ArchiveAPI:
+    """Class handling the archiving of sample SPRING files."""
+
+    def __init__(
+        self, ddn_dataflow_api: DDNDataFlowApi, housekeeper_api: HousekeeperAPI, status_db: Store
+    ):
+        self.housekeeper_api: HousekeeperAPI = housekeeper_api
+        self.ddn_api: DDNDataFlowApi = ddn_dataflow_api
```

Comment:
I can't see that this one is used anywhere, but I'm guessing it is in a separate PR

Reply:
Indeed, good catch. Removing in this PR